gateway/simulator/simulation_utils.go

```diff
@@ -18,7 +18,7 @@ import (
 type PlayBackMockConf struct {
 	BareDomain      string `json:"bare_domain"`
 	AccessKeyID     string `json:"access_key_id"`
-	AccessSecretKey string `json:"access_secret_Key"`
+	SecretAccessKey string `json:"secret_access_key"`
```

Comment:
This json is coming from a pre-made file. To avoid changing it, let's bring back the "access_secret_key" as the json tag (the field name SecretAccessKey is ok)